doc/source/data/inspecting-data.rst

```diff
@@ -11,6 +11,7 @@ This guide shows you how to:
 * `Describe datasets <#describing-datasets>`_
 * `Inspect rows <#inspecting-rows>`_
 * `Inspect batches <#inspecting-batches>`_
+* `Inspect Dataset runtime statistics <#inspecting-stats>`_
```

Comment:
```suggestion
* `Inspect runtime statistics <#inspecting-stats>`_
```

Do we need "Dataset" here? "Inspect Dataset runtime statistics" is more verbose than the other bullet points

Reply:
updated